common/src/main/scala/org/apache/celeborn/common/util/Utils.scala

```diff
@@ -499,9 +499,6 @@ object Utils extends Logging {
     val numThreads = defaultNumThreads(numUsableCores)
     conf.setIfMissing(s"celeborn.$module.io.clientThreads", numThreads.toString)
     conf.setIfMissing(s"celeborn.$module.io.clientThreads", numThreads.toString)
-    // TODO remove after releasing 0.2.0
-    conf.setIfMissing(s"rss.$module.io.serverThreads", numThreads.toString)
-    conf.setIfMissing(s"rss.$module.io.clientThreads", numThreads.toString)
```

Comment:
is this PR for 0.2? if yes, we should keep this for compatibility

Reply:
ok